pkgs/development/python-modules/linode-api/default.nix

```diff
@@ -15,11 +15,16 @@ buildPythonPackage rec {
 
   disabled = (pythonOlder "2.7");
 
-  buildInputs = [];
   propagatedBuildInputs = [ requests ]
                              ++ stdenv.lib.optionals (!isPy3k) [ future ]
                              ++ stdenv.lib.optionals (pythonOlder "3.4") [ enum34 ];
 
+  postPatch = (stdenv.lib.optionalString (isPy3k) ''
+    sed -i -e '/"future",/d' setup.py
+  '') + (stdenv.lib.optionalString (!pythonOlder "3.4") ''
+    sed -i -e '/"enum34",/d' setup.py
+  '');
```

Comment:
The tests for python 3 still fail, while they are not executed for python2 (try building `pythonPackages3.linode-api`).